singularity-eos/eos/eos_spiner_rho_sie.hpp

```diff
@@ -308,9 +354,11 @@ class SpinerEOSDependsRhoSie : public EosBase<SpinerEOSDependsRhoSie> {
   static constexpr const int _n_lambda = 1;
   static constexpr const char *_lambda_names[1] = {"log(rho)"};
   DataStatus memoryStatus_ = DataStatus::Deallocated;
+  TransformData Transform_data_;
```

Comment:
```suggestion
  TransformDataT Transform_data_;
```

Reply:
It's more C++ style to do `TransformData_t`. Don't ask me why. Ancient idiosyncracy.